src/core/qgis.h

```diff
@@ -2072,6 +2072,48 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( ArcGisRestServiceType )
 
+    /**
+     * Relationship types.
+     *
+     * \note Prior to QGIS 3.28 this was available as QgsRelation::RelationType.
+     *
+     * \since QGIS 3.28
+     */
+    enum class RelationshipType SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsRelation, RelationType ) : int
+      {
```

Comment:
Weird that the auto-formatter didn't catch this one.

```suggestion
{
```

Reply:
Ah, it always does that when the monkey patch macro is in use. Not much we can do there...